www-servers/apache/apache-2.4.33.recipe

```diff
@@ -0,0 +1,309 @@
+	lib:mod_reqtimeout$secondaryArchSuffix
+	lib:mod_request$secondaryArchSuffix
+	lib:mod_rewrite$secondaryArchSuffix
+	lib:mod_sed$secondaryArchSuffix
+	lib:mod_session$secondaryArchSuffix
+	lib:mod_session_cookie$secondaryArchSuffix
+	lib:mod_session_dbd$secondaryArchSuffix
+	lib:mod_setenvif$secondaryArchSuffix
+	lib:mod_slotmem_shm$secondaryArchSuffix
+	lib:mod_socache_dbm$secondaryArchSuffix
+	lib:mod_socache_memcache$secondaryArchSuffix
+	lib:mod_socache_shmcb$secondaryArchSuffix
+	lib:mod_speling$secondaryArchSuffix
+	lib:mod_ssl$secondaryArchSuffix
+	lib:mod_status$secondaryArchSuffix
+	lib:mod_substitute$secondaryArchSuffix
+	lib:mod_unique_id$secondaryArchSuffix
+	lib:mod_unixd$secondaryArchSuffix
+	lib:mod_userdir$secondaryArchSuffix
+	lib:mod_version$secondaryArchSuffix
+	lib:mod_vhost_alias$secondaryArchSuffix
+	lib:mod_watchdog$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libapr_1$secondaryArchSuffix
+	lib:libaprutil_1$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+#	lib:libldap$secondaryArchSuffix
+	lib:libpcre$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+"
+
+GLOBAL_WRITABLE_FILES="
+	settings/extra/httpd-autoindex.conf keep-old
+	settings/extra/httpd-dav.conf keep-old
+	settings/extra/httpd-default.conf keep-old
+	settings/extra/httpd-info.conf keep-old
+	settings/extra/httpd-languages.conf keep-old
+	settings/extra/httpd-manual.conf keep-old
+	settings/extra/httpd-mpm.conf keep-old
+	settings/extra/httpd-multilang-errordoc.conf keep-old
+	settings/extra/httpd-ssl.conf keep-old
+	settings/extra/httpd-userdir.conf keep-old
+	settings/extra/httpd-vhosts.conf keep-old
+	settings/extra/proxy-html.conf keep-old
+	settings/httpd.conf keep-old
+	settings/magic keep-old
+	settings/mime.types keep-old
+	settings/original/extra/httpd-autoindex.conf keep-old
+	settings/original/extra/httpd-dav.conf keep-old
+	settings/original/extra/httpd-default.conf keep-old
+	settings/original/extra/httpd-info.conf keep-old
+	settings/original/extra/httpd-languages.conf keep-old
+	settings/original/extra/httpd-manual.conf keep-old
+	settings/original/extra/httpd-mpm.conf keep-old
+	settings/original/extra/httpd-multilang-errordoc.conf keep-old
+	settings/original/extra/httpd-ssl.conf keep-old
+	settings/original/extra/httpd-userdir.conf keep-old
+	settings/original/extra/httpd-vhosts.conf keep-old
+	settings/original/extra/proxy-html.conf keep-old
+	settings/original/httpd.conf keep-old
+"
+
+BUILD_REQUIRES="
+	devel:libapr_1$secondaryArchSuffix
+	devel:libaprutil_1$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:libexpat$secondaryArchSuffix
+	devel:libiconv$secondaryArchSuffix
+#	devel:libldap$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:autoconf
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize
```

Comment:
Oops, I just noticed I missed this one :)
```diff
-	cmd:libtoolize
+	cmd:libtoolize$secondaryArchSuffix
```

BTW, regarding the previous recipe (`2.4.29`), don't you want to remove it?
If `2.4.33` works, then we don't need `2.4.29`.
And if `2.4.33` does not work, then it would be better to fix it before merging.

In other words, it would be nice to see if this release is working fine, even with a minimalistic config.
(If it looks OK then we can drop the previous recipe. If not, then we can't merge this as is.)